README.md

```diff
@@ -123,7 +123,7 @@ After installing OSM, [onboard a microservice application](docs/onboard_services
 
 ## Demo and Examples
 
-The [automated demo](demo/README.md) shows how OSM can manage, secure and provide observability for microservice environments.
+The [automated demo](demo/README.md) is a set of scripts anyone can run and shows how OSM can manage, secure and provide observability for microservice environments.
```

Comment:
`is a set of scripts anyone can run` is this even necessary?

Reply:
Maybe just `automated demo scripts`?

Reply:
When I saw "automated demo", I realized people may not know what even means. I thought I'd clarify it is a set of scripts. People might think it's a video.